Home.md

```diff
@@ -14,4 +14,4 @@ If you are interested in working on Oppia's Android app, you should also take a
 
 If you run into any problems, you can check out the [user documentation](http://oppia.github.io/), our [GitHub Discussions board](https://github.com/oppia/oppia/discussions), or post to our [developer mailing list](https://groups.google.com/forum/?fromgroups#!forum/oppia-dev).
 
-Finally, if, after reading the above links, you'd like to help, but aren't sure how -- don't worry! The Oppia project is very multi-faceted, and we'd be glad to help you find something to do that matches your interests and skills. Just fill out [this form](https://forms.gle/jEytndtgdsx7BrnV6) to let us know what you'd like to help with, or write to us at [volunteer@oppia.org](mailto:volunteer@oppia.org) and tell us a bit about yourself and what you'd like to do. We'll do our best to help you get started!
+Finally, if, after reading the above links, you'd like to help, but aren't sure how -- don't worry! The Oppia project is very multi-faceted, and we'd be glad to help you find something to do that matches your interests and skills. Just fill out [this form](https://forms.gle/jEytndtgdsx7BrnV6) to let us know what you'd like to help with, or write to us at [volunteer@oppia.org](mailto:volunteer@oppia.org) and tell us a bit about yourself and what you'd like to do. We'll do our best to help you get started! If you run into any problems along the way, we're here to help! Check out our [wiki page on getting help](https://github.com/oppia/oppia/wiki/Get-help) for the communication channels you can use.
```

Comment:
Drop the last sentence here, it's not really valid in this context. The "get help" stuff is for coding volunteers and this paragraph is for those who aren't coders.

Reply:
done